lua/wikis/ageofempires/Infobox/Map/Custom.lua

```diff
@@ -62,6 +63,12 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+---@param args table
+---@return string[]
+function CustomMap:getGameModes(args)
+	return {self:_getType(args.type)}
+end
+
```

Comment:
fwiw map type is not related to game mode in AoE (at least not for the types currently supported).
I'd rather leave this empty for now.